src/styles/search.css

```diff
@@ -0,0 +1,31 @@
+@import url("https://fonts.googleapis.com/css2?family=Trispace:wght@500&display=swap");
+
+.search-input {
+  margin: auto;
+  width: 20%;
+  height: 50px;
+  border-radius: 11px;
+  font-size: 20px;
+  padding: 0 10px;
+  font-family: "Trispace", sans-serif;
+  background-color: rgba(167, 187, 187, 0.57);
+}
+.search-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.search-btn {
+  margin: auto;
+  width: 20%;
```

Comment:
Generally Id keep the width of things like buttons fixed, or at least within a small range of sizes depending on screen size